src/components/Item/ItemCard.tsx

```diff
@@ -6,15 +6,10 @@ import Vote from '@/components/Vote/Vote';
 import { IMAGE_PATH, DEFAULT_IMAGE } from '@/constants/constantValues';
 import { MoviesShowsContext } from '@/context/Context';
 import calculateAverageVote from '@/helpers/calculateAverageVote';
-import { ItemType } from '@/types/types';
+import { ItemCardProps } from '@/types/types';
```

Comment:
Let's use the following convention in writing TS types. TS types specific to the component should be added to the components folder in the `types.ts(x)` file. All common types, shared by multiple features in the codebase should be added to the project's root types.